clients/pkg/promtail/server/server.go

```diff
@@ -51,6 +54,8 @@ type Config struct {
 	ExternalURL       string `yaml:"external_url"`
 	HealthCheckTarget *bool  `yaml:"health_check_target"`
 	Disable           bool   `yaml:"disable"`
+	Reload            bool   `yaml:"reload"`
```

Comment:
We should consider naming this more clearly.

I would suggest `enable_runtime_reload`

Reply:
done